specification/search/data-plane/Microsoft.Azure.Search.Service/stable/2019-05-06/examples/SearchServiceCreateOrUpdateSkillset.json

```diff
@@ -84,6 +84,31 @@
               "targetName": "keyPhrases"
             }
           ]
+        },
+        {
+          "@odata.type": "#Microsoft.Skills.Custom.WebApiSkill",
```

Comment:
Is `Custom` really part of this namespace? Was it always that way? Are there any other classes in the `Custom` namespace?

Reply:
Yup, this was always this way and so far there is only this one skill in this namespace